class.jetpack-gutenberg.php

```diff
@@ -499,4 +499,24 @@ public static function enqueue_block_editor_assets() {
 
 		wp_enqueue_style( 'jetpack-blocks-editor', $editor_style, array(), $version );
 	}
+
+	/**
+	 * CSS class names from a block's type and attributes
+	 *
+	 * @param string $type - Block type.
+	 * @param array  $attr - Array containing the map block attributes.
+	 *
+	 * @return string
+	 */
+	public static function block_classes( $type, $attr ) {
+		$align   = isset( $attr['align'] ) ? $attr['align'] : 'center';
```

Comment:
😬 I don't think it's safe to fall back to `center` here. If `align` is unset, we don't know what to do at this level.

We'd need to handle the default alignment where this is invoked.

Reply:
Hmmm. Could I add `$default_alignment` as a third param, defaulting to `center`?

Reply:
🤔 Maybe something like this?

```php
	public static function block_classes( $type, $attr, $default_attrs = array() ) {
```

Reply:
Ah, interesting, you're suggesting a broader solution, where any defaults can be passed in, and we'd basically merge `$attr` and `$default_attrs`? If we do that, this starts to get close to the open question of duplicated attribute defaults in Javascript & PHP that's come up recently. I have slight concern that adding a partial solution here might be premature since the issue hasn't been fully hashed out yet?

Reply:
Yes, that might be a not-too-distant idea we could set-up here 😉 

The other reason is that I'm hesitant to add a `$default_alignment` 3rd positional parameter. I suspect we'd need both default class and default alignment for some blocks (tiled gallery uses styles and relies on the class for this). My intuition says that `$default_attributes` could cover most of our needs without needing to continually add more positional parameters, but it's just intuition at this point 🙂

Reply:
After some off-GitHub conversation, we've decided to roll back this part of the PR and open a dedicated PR dedicated to this change.

Reply:
Rolled back in https://github.com/Automattic/jetpack/pull/11092/commits/00f297c75c43ba23b5eb471b2979cb47b788e606